src/install/mongodb/host/percona.mdx

```diff
@@ -0,0 +1,31 @@
+---
+componentType: default
+headingText: Prepare MongoDB or Percona
+---
+
+<Callout variant="tip">
+    Complete these steps on the mongos to be monitored. If you want to collect mongod-level metrics,
+    such as host or replica set statistics, create the role
+    and user on each mongod as well.
```

Comment:
```suggestion
    Complete these steps on the mongos to be monitored. If you want to collect mongod-level metrics,
    such as host or replica set statistics, create the role
    and user on each mongod as well.
```

Are mongos and mongod typos? Just want to double check. Perhaps they should be uppercased?